scripts/seed-wechat-accounts.py

```
@@ -0,0 +1,220 @@
+    )
+
+    # Only register the channel in openclaw.json when wechat is enabled for
+    # this build. When the operator stopped the channel before rebuild,
+    # NEMOCLAW_MESSAGING_CHANNELS_B64 omits "wechat" and we leave the patch
+    # off — the account state files above are still on disk and ready for a
+    # later `channels start wechat` rebuild to activate.
+    if _wechat_enabled():
+        _patch_openclaw_config(account_id)
+    else:
+        print(
+            f"[seed-wechat-accounts] wechat not in active channels; preserving account "
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)